.github/workflows/precommit.yml

```diff
@@ -302,6 +302,115 @@ jobs:
             .tox/.coverage
             .tox/coverage.xml
 
+  test-snowpark-pandas:
```

Comment:
do we plan to configure that if only snowpark pandas files change, we only run snowpark pandas tests?

Reply:
Yep will happen in a follow-up PR!